src/ocean/util/cipher/gcrypt/core/Gcrypt.d

```diff
@@ -352,8 +353,6 @@ private class GcryptBase ( Algorithm algorithm, Mode mode )
 
     version ( UnitTest )
     {
-        import ocean.core.Test;
```

Comment:
Doesn't seem to be directly related to the comment description. Is the change in wrong commit or I am missing the context?